ml-testing-toolkit-cli/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: mojaloop/ml-testing-toolkit
-  tag: v14.0.4
+  repository: mojaloop/ml-testing-toolkit-client-lib
```

Comment:
Nothing we necessarily need to change right now, but this is not the greatest name for a docker image.

Lib does not imply a CLI or something like that, I feel like we name much better, i.e.

```
  repository: mojaloop/ml-testing-toolkit-client-cli
```

Something to consider in future.

Reply:
Sure